specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkWatcher.json

```diff
@@ -2727,6 +2970,81 @@
         }
       }
     },
+    "FlowLogListResult": {
+      "properties": {
+        "value": {
```

Comment:
Include 'nextLink' even if you just set it to null. This allows you to add it later without an API breaking change.

Reply:
Done